templates/multisig/contracts/FundraisingMultisigTemplate.sol

```diff
@@ -331,7 +331,7 @@ contract FundraisingMultisigTemplate is EtherTokenConstant, BaseTemplate {
         (,,,, Controller controller) = _fundraisingAppsCache();
 
         // create and grant ADD_PROTECTED_TOKEN_ROLE to this template
```

Comment:
minor: comment mentions `ADD_PROTECTED_TOKEN_ROLE` -> should be `ADD_COLLATERAL_TOKEN_ROLE`